packages/flutter/test/widgets/shape_decoration_test.dart

```diff
@@ -94,10 +94,7 @@ Future<void> main() async {
     );
   });
 
-  testWidgets('TestBorder and Directionality - 2',
-  // TODO(polina-c): dispose ImageStreamCompleterHandle, https://github.com/flutter/flutter/issues/145599 [leaks-to-clean]
```

Comment:
This PR doesn't include a fix for this. Was this fixed in another PR, which just forgot to update these tests?